tests/resources.py

```diff
@@ -20,6 +20,10 @@ def find(name: str) -> str:
     return os.path.normpath(os.path.join(os.path.dirname(__file__), DATA_DIR, name))
 
 
+def find_in_testdata(name: str) -> Path:
```

Comment:
```suggestion
def construct_absolute_path_in_testdata(name: str) -> Path:
```
For me the verb "find" imply searching for it by walking the directory tree. I would expect the give "foo.py" and get back the file in ``testdata/bar/`` called ``foo.py``

Reply:
I changed it to use `get_testdata_path`. The other one is a bit long.